benchmarks/benchmarks/signal_filtering.py

```diff
@@ -102,3 +102,16 @@ def time_medfilt2d(self, threads):
 
     def peakmem_medfilt2d(self, threads):
         self._medfilt2d(threads)
+
+
+class FreqzRfft(Benchmark):
+    param_names = ['worN']
+    params = [
+        [64, 65, 128, 129, 256, 257, 512, 513],
+    ]
+
+    def setup(self, worN):
+        self.y = np.random.RandomState(0).randn(worN,10000)[...,np.newaxis]
```

Comment:
I don't think you should need the `10000`, I think in theory `asv` should pick the right number of repetitions to use (and the contents/values shouldn't actually matter in this case, so varying them doesn't have much benefit).

Reply:
There is a noticeable difference between vectorized and not_vectorized input. I added a parameter to benchmark both cases.